histogram.js

```diff
@@ -16,12 +17,16 @@ function genHtml(styles) {
       }
 
       .iobio-histogram {
+        display: flex;
+        align-items: center;
+        justify-content: center;
         width: 100%;
         height: 100%;
       }
 
       .iobio-histogram-title {
```

Comment:
I think we can remove titles from the charts now that we're using the info button components for that, right?

Reply:
Yes, I was about to ask you that. We can get rid of them if they're no longer an option for the user.